### DISCUSSION TITLE
Переделать мигалки для экстренных служб

### DISCUSSION THREAD

**Discussion by octothorpteam** [octothorp-team/public-dobrograd#591] [Предложения]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/billblack/32/763_2.png) [BillBlack](https://forum.octothorp.team/u/BillBlack) ([#4278](https://forum.octothorp.team/p/4278))

Итак. Суть.
 Сделать стандартные  красно-синие мигалки для машин полиции, ибо мигалки из гта 4, где за прототип взяты машины полиции NY со специфичными мигалками малость не вписываются в сеттинг Доброграда.

Как по мне, вот такие мигалки ([https://i.imgur.com/0SEJZaP.jpg](https://i.imgur.com/0SEJZaP.jpg)) лучше, чем красные.

Это лично мое мнение. У тех машин, что были из GTA:SA были плохие мигалки, но правильные, а хорошие были у тех, что самые первые, те, уютные маленькие машинки. Вот это были красивые маячки.
 Вот чем-то подобным красивым, и обязательно сине-красным заменить то нью-йоркское чудо, что унаследовали все экстренные службы Доброграда. И не просто сделать вместо белых синие, а именно заменить вот это вот на светодиодные балки (или плафоны с вращающимися зеркалами).

- Да, сине-красные маячки действительно смотрятся красивее, чем эти мигалки, давайте их заменим на сине-красные, так будет лучше;
- Нет, данные мигалки очень хорошие и мы не ориентируемся на какой-то там Лос-Анджелес, мы особенный город, больше ни у кого, кроме нас таких нет.

0голосов

P.S Я не пытался написать несерьезно, если что. Если есть претензии по опросу, что написан слишком смешно и неправильно - извините. Не нес цели сделать неправильный опрос.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/_r.u.s.t.i.k/32/341_2.png) [_R.U.S.T.I.K](https://forum.octothorp.team/u/_R.U.S.T.I.K) ([#4562](https://forum.octothorp.team/p/4562))


Да это дейстительно правильно было бы, по скольку не просто так сделали красно синие маяки, а именно что бы показывало приемущество на дороге.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/jusko/32/2001_2.png) [jusko](https://forum.octothorp.team/u/jusko) ([#4611](https://forum.octothorp.team/p/4611))


Простое, приятное косметическое предложение. Создание будет несложным, а игрокам будет хоть немного покомфортнее и приятнее глазу при виде мигалок общепринятого цвета. Я за.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/chelog/32/9_2.png) [chelog](https://forum.octothorp.team/u/chelog) ([#4698](https://forum.octothorp.team/p/4698))


Вот к этому есть претензия в опросе, опять же, стилистическая окраска склоняет не выбирать этот ответ, это некруто

![](https://forum.octothorp.team/user_avatar/forum.octothorp.team/billblack/40/763_2.png) BillBlack:


> данные мигалки очень хорошие и мы не ориентируемся на какой-то там Лос-Анджелес, мы особенный город, больше ни у кого, кроме нас таких нет

* * *

Несмотря на это я считаю, что сине-красные лучше смотрятся, если ролевые менеджеры дадут добро, закинем задачку нашим 3D-шникам

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/billblack/32/763_2.png) [BillBlack](https://forum.octothorp.team/u/BillBlack) ([#4717](https://forum.octothorp.team/p/4717))


Да, я понял свою ошибку, но снизу сказал, что не имел цели сделать такой опрос.
 Кроме того, смысл не несет в себе “это выбирать не надо, потому что мы особенные какие-то, а особенные - значит что-то обидное, мол только у кого-то, а тут мы слизали”. Нет, здесь есть смысл, что мы мол, особенный город, у нас вот такие маячки, а больше таких нигде нет, поэтому давайте их оставим". Если здесь можно найти негативный или агитирующий вариант, то только, если у человека достаточно богатое воображение. Кроме того, не всегда такая стилистическая окраска дает эффект нужного выбора. Далеко не всегда.

**Comment by LENFYS** [comment]
Уже сделали
